### DISCUSSION TITLE
Microfrontend multiple dialogs

### DISCUSSION THREAD

**Discussion by PaoloDiGallo** [tailwindlabs/headlessui#2777] [Help]
Hello, i recently started developing a microfrontend application for work and i encountered a problem with opening multiple dialogs.

i have an application that is called ConfirmModals - a Vue app that listens to an event and opens a modal to confirm or cancle a user operation.

than i have a react application that is a modal to show the details of an event the user created and here the user can delete/edit this event - on the delete we want to open the first modal - ConfirmModal - that is on another application

this means that since this 2 apps are not nested the Dialogs are opened at the same level and when one is closed the other is closed as well. This is not what we wanted as a user might have pressed it by mistake, may want to go back and continue editing or whatever

is there any way to have the dialog open as nested in the other one instead of as a sibling?
for now we are closing the first, opening the second, closing the second and if needed re-opening the first - it works but it's not optimal and quite ugly to see if i am being honest

**Comment by ManuelcsNuant** [comment]
I am having the same issue, @PaoloDiGallo did you manage to solve this one?

**Reply by PaoloDiGallo** [reply]
no luck in headless ui
had to remove headless's dialogs and made my own webComponent to handle all the instances (didn't want to have some with headless and some with our own custom)
